packages/std/src/amp/messages.rs

```diff
@@ -298,6 +298,7 @@ pub struct AMPPkt {
     /// Any messages associated with the packet
     pub messages: Vec<AMPMsg>,
     pub ctx: AMPCtx,
+    pub id: Option<String>,
```

Comment:
https://github.com/andromedaprotocol/andromeda-core/blob/55b69b0b572217166f6b6f088ec54973c39f2cea/packages/std/src/amp/messages.rs#L214

AMPCtx already has an ID field. We could convert this to be a string instead.